### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by antipyrrhus**
I've been using the remove_all_labels() method pretty extensively during unit testing and was always bothered by the spelling error in 'Droping'.

### CODE REVIEW

**Comment by aanastasiou** [comment]
@antipyrrhus Thats great, thanks for pointing that out.

File: neomodel/relationship_manager.py [hunk 1]
```diff
@@ -73,6 +73,34 @@ def connect(self, node, properties=None):
                 "is no longer supported."
             )
 
+        # Check the other node's constraints
+        reltype = self.definition['relation_type']
```
Comment (alimcmaster1) [inline neomodel/relationship_manager.py:77]: This looks to be outside of the scope of the PR - mind removing @antipyrrhus

**Comment by whatSocks** [comment]
commits cherry-picked and merged via https://github.com/neo4j-contrib/neomodel/pull/593
